test/registerProcessor_test.js

```diff
@@ -0,0 +1,244 @@
+goog.module('datalayerhelper.helper.testing.registerProcessor');
+goog.setTestOnly();
+
+const {DataLayerHelper} = goog.require('helper');
+
+describe('The registerProcessor method of helper', () => {
+  let dataLayer;
+  let dlh;
```

Comment:
Might want to consider renaming this one: https://google.github.io/styleguide/jsguide.html#naming

Reply:
Changed to dataLayerHelper